src/resource/saveSolidDatasetInContainer.ts

```diff
@@ -27,18 +27,14 @@ import {
 import { VerifiableCredential } from "@inrupt/solid-client-vc";
 import { fetchWithVc } from "../fetch";
 import { FetchOptions } from "../type/FetchOptions";
-
-// TODO: This should come from @inrupt/solid-client
-interface SaveInContainerOptions extends FetchOptions {
-  slugSuggestion?: string;
-}
+import { SaveInContainerOptions } from "../type/SaveInContainerOptions";
 
 /**
  * Given a SolidDataset, store it in a Solid Pod as a new Resource inside a Container.
  *
  * The Container at the given URL should already exist; if it does not, you can initialise it first
- * using [[createContainerAt]], or directly save the SolidDataset at the desired location using
- * [[saveSolidDatasetAt]].
+ * using createContainerAt, or directly save the SolidDataset at the desired location using
+ * {@link resource.saveSolidDatasetAt}.
```

Comment:
Good catch!

Reply:
Huh! Is that how you do links within the one piece of documentation in typedoc? Neat.